### PR TITLE
Treat translations as XML, not text

### PR DESCRIPTION
**Pull request by kaapstorm**
For review, in case there's anything useful in here.

I took what Jenny had worked on, and saw whether I could export the `<output>` tags, and verify just the translations that may contain `<output>` tags. But I didn't go through the tests carefully enough, and I have failures that I didn't get to resolve. They may be simple problems.

Anyway, use any of it if you can, throw it away if you can't.

[FB 169204](http://manage.dimagi.com/default.asp?169204)

@emord, @snopoke, @orangejenny

### CODE REVIEW

File: corehq/apps/app_manager/translations.py [hunk 1]
```diff
@@ -446,6 +452,10 @@ def update_form_translations(sheet, rows, missing_cols, app):
     :return:  Returns a list of message tuples. The first item in each tuple is
     a function like django.contrib.messages.error, and the second is a string.
     """
+    def value_is_safe(value_element):
+        is_safe = lambda el: isinstance(el, etree.Entity) or isinstance(el, etree.Element) and el.tag in SAFE_TAGS
```
Comment (emord) [inline corehq/apps/app_manager/translations.py]: These aren't actually classes. We use lxml here which uses `etree._Entity` and `etree._Element`. I'm not 100% sure on the differences between those

**Comment by emord** [comment]
I started work on this one here: https://github.com/dimagi/commcare-hq/pull/7182/

I like that your solution seems more extendable as I'm explicitly using a regex to escape output values in mine. 

Also I saw that yours converts 
`Blah < blah <output value="/data/question4"/> blha` to 
`blah <output value="/data/question4"/> blha<output value="/data/question4"/> blha`

File: corehq/apps/app_manager/translations.py (hunk 1 above)
Comment (kaapstorm) [inline corehq/apps/app_manager/translations.py]: Ah! Thank you. (An entity is like `&#234;`, e.g.

    >>> etree.Entity("#234")
    &#234;

)

**Comment by kaapstorm** [comment]
Ha! Interesting. I'd have thought `Blah < blah` would all have been the text of the opening element.

**Comment by czue** [comment]
is this PR still relevant?

**Comment by emord** [comment]
I don't think it is. Closing.

**Pull request closed without merging**